From 8afdd8babb9e82bcfde78e851b24b698436a6f9f Mon Sep 17 00:00:00 2001
Message-Id: <8afdd8babb9e82bcfde78e851b24b698436a6f9f.1495466847.git.baruch@tkos.co.il>
From: Baruch Siach <baruch@tkos.co.il>
Date: Mon, 22 May 2017 15:42:49 +0300
Subject: [PATCH] ip: include libc headers first

Including libc headers first helps as a workaround to redefinition of struct
ethhdr with a suitably patched musl libc that suppresses the kernel
if_ether.h.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Upstream status: posted http://marc.info/?l=linux-netdev&m=149545968900803&w=2

 ip/iplink_bridge.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ip/iplink_bridge.c b/ip/iplink_bridge.c
index 818b43c89b5b..cccdec1c203a 100644
--- a/ip/iplink_bridge.c
+++ b/ip/iplink_bridge.c
@@ -13,9 +13,9 @@
 #include <stdlib.h>
 #include <string.h>
 #include <netinet/in.h>
+#include <netinet/ether.h>
 #include <linux/if_link.h>
 #include <linux/if_bridge.h>
-#include <netinet/ether.h>
 #include <net/if.h>
 
 #include "rt_names.h"
-- 
2.11.0

